docs/CI.md

```diff
@@ -30,24 +30,39 @@ More specifically, the following workflows are included:
 
 <sup>1</sup> On-demand workflows are triggered manually from the GitHub Actions tab. Each GitHub organization/individual has a quota on storage on private repositories. Uploading artifacts counts against your quota. You may consider configuring your workflows to only upload what you need. You can learn more about GitHub's storage limits in their [official documentation](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions).
 
-## Github Actions for firebase hosting
+### Download your bundled executable to install and run your task
 
-Automatic deployment on a firebase hosting site can also be setup with github actions. Before proceeding with the follwing steps, please make sure that firebase is configured by following the [Getting started with firebase section under Set Up Firebase](firebase.md)
+You can download the executable file from either the tagged release page or the GitHub actions page. To install, unzip the downloaded file, install and run.
 
-- Run `firebase init hosting:github` in the terminal and follow the CLI prompts to start setting up the github actions.
-  - For the github directory question, select your task github directory.
-  - When prompted with the npm command, type `npm install && npm run build:firebase`
+The executable does not require installation of any additional software.
 
-There should be two new `.yml` files, `firebase-hosting-pull-request.yml` and `firebase-hosting-merge.yml`, created in the `.github/workflows` directory for a deployment preview for each pull request and the official deployment when merged onto the main branch.
+### Uninstall the task
```

Comment:
Perhaps add in why we're uninstalling the task?

Reply:
This is legacy and not something we've touched at all for the workshop. Is it okay to leave it? I'm not entirely sure what this is supposed to be for